src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add: function(num) {
```

Comment:
```suggestion
    add(num) {
```
if you creating a method in the object, you don't need to use function keyword, use shortcut instead 👍